webapp/src/components/calculations/options.tsx

```diff
@@ -191,7 +191,7 @@ const CalculationOptions = (props: BaseCalculationOptionProps): JSX.Element => {
             isSearchable={false}
             components={{DropdownIndicator, ...(props.components || {})}}
             defaultMenuIsOpen={props.menuOpen}
-            autoFocus={true}
+            autoFocus={false}
```

Comment:
I'm not 100% sure if this is affecting other expected behavior, but otherwise looks good to me. @ogi-m can you confirm that the calculation options works as expected around the focus handling?